pkg/operator2/configsync.go

```diff
@@ -121,48 +126,67 @@ func newSourceDataIDPConfigMap(index int, configMap configv1.ConfigMapNameRefere
 	return dest, ret
 }
 
-func newIDPSyncData() idpSyncData {
-	configMaps := map[string]sourceData{}
-	secrets := map[string]sourceData{}
+func newSourceDataTemplateSecret(secretRef configv1.SecretNameReference, key string) (string, sourceData) {
+	dest := getTemplateName(secretRef.Name, key)
+
+	vol, mount, path := secretVolume(getTemplatePath("template", dest), dest, key)
```

Comment:
nit: bit awkward, maybe put a `path := getTem...` below the `dest` var declaration

Reply:
fixed